core/src/main/java/org/axonframework/eventhandling/ReplayToken.java

```diff
@@ -39,6 +41,60 @@
     @JsonTypeInfo(use = JsonTypeInfo.Id.CLASS, include = JsonTypeInfo.As.PROPERTY, property = "@class")
     private final TrackingToken currentToken;
 
+    /**
+     * Creates a new TrackingToken that represents the given {@code startPosition} of a stream, in reset state,
+     * when appropriate.
+     *
+     * @param tokenAtReset  The token present when the reset was triggered
+     * @param startPosition The position where the token should be reset to
+     * @return A token that represents a reset to the tail of the stream
+     */
+    public static TrackingToken createReplayToken(TrackingToken tokenAtReset, TrackingToken startPosition) {
+        if (tokenAtReset == null) {
+            // the tokenAtReset is not (completely) past startposition. Reset may not be used to advance tokens
```

Comment:
I'm not sure what you mean with this comment. Do you mind elaborating?

Reply:
It means that if you wish to "reset" a processor to a position that could cause you to miss certain events, it won't do it.
Thinking of it.... why would we care?